src/display/displayCommon.h

```diff
@@ -238,7 +238,7 @@ void displayLogo(String displaymessagetext, String displaymessagetext2) {
  * @brief display shot timer
  */
 void displayShottimer(void) {
-    if ((machineState == kBrew || brewSwitchState == kBrewSwitchFlushOff) && SHOTTIMER_TYPE == 1) {
+    if ((machineState == kBrew || brewSwitchState == kBrewSwitchFlushOff) && FEATURE_SHOTTIMER == 1 && SHOTTIMER_TYPE == 1) {
```

Comment:
I think this got broken during your rebase, theree was supposed to be a 
`if (FEATURE_SHOTTIMER == 0) {
    return;
}` at the top of displayShottimer, then these can be removed

Reply:
urgs, this was my fault, I´m sorry!
Fixed and rebased again, ready for merge.